website/technischeumsetzung.md

```diff
@@ -1,17 +1,203 @@
 ---
-title: lorem-ipsum
-date: '2021-06-09'
-layout:
-  label: page
-  value: page
-menu:
-  label: Main Menu
-  value: main
-bodyClass: lorem-ipsum
+title: Technische Umsetzung
+date: '2018-11-28T15:14:39+10:00'
+layout: default_5
+intro_image: null
+intro_image_absolute: false
+intro_image_hide_on_mobile: true
 ---
-## Lorem ipsum
 
-Lorem ipsum dolor sit amet, **consectetur adipiscing elit**, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+Anhand eines Modellnamens werden im *ilidata.xml* Pfade zu *Metakonfigurationsfiles* gefunden. Diese *Metakonfigurationsfiles* enthalten, neben Konfigurationsparameter, *DatasetMetadata-Ids*. Anhand dieser *DatasetMetadata-Ids* werden im *ilidata.xml* die Pfade zu den *Toppingfiles* gefunden.
 
-- Lorem ipsum
-- dolor sit amet
\ No newline at end of file
+> Andere Tools verwenden einen anderen Workflow. So wird beispielsweise dem *ili2db* das *Metakonfigurationsfile* übergeben. Dort ist in der *Metakonfiguration* dann auch das Modell definert. 
+
+![uml](../images/uml_modelbaker.png)
```

Comment:
```suggestion
![uml](_services/images/uml_modelbaker.png)
```
Or better move the image uml_modelbaker.png into the main images folder (or move technischeumsetzung.md) into the _services folder...